ai_summarizer/models.py

```
@@ -30,5 +30,14 @@ def clean(self) -> None:
                 })
 
     def __str__(self) -> str:
-        """نمایش کوتاه برای ادمین: 'AI Summary for <Patient> - <Model>'."""
-        return f"AI Summary for {self.patient.full_name} - {self.content_type.model}"
\ No newline at end of file
+        """
+        یک رشتهٔ نمایش خوانا برای پنل ادمین که خلاصهٔ AI را به صورت "AI Summary for <نام بیمار> - <نام مدل>" ارائه می‌دهد.
```

Ruff (0.12.2)

Line too long (119 > 88)

(E501)